resources/definitions/bel_all_metal_delta.def.json

```diff
@@ -0,0 +1,94 @@
+{
+    "version":      2,
+    "name":         "Blue Eagle Labs All Metal Delta",
```

Comment:
We normally leave the manufacturer out of this, if the manufacturer doesn't mention it as part of the name either.

From what I can tell from https://www.blueeaglelabs.com/, the name of this printer is "Metal Delta". It doesn't have "All" in the name anywhere I could find.